python-iml-manager.spec.in

```diff
@@ -34,7 +34,9 @@ Source11:       iml-power-control.service
 Source12:       iml-settings-populator.service
```

Comment:
Maybe in another patch we can update the repo to use the new copr deploy method, which should no longer require the spec.in file (and `includes` directory of course).

Reply:
yeah. The rust-iml.spec in this patch does use copr builder.